docs/application/native/guides/internationalization/i18n.md

```diff
@@ -3405,6 +3480,75 @@ To manage plural formatting:
                                     BUFSIZE, output, &output_length);
     ```
 
+<a name="date_interval_examples"></a>
+## Manage Date Intervals
+
+From Tizen 5.0, you can use the DateInterval API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) applications) and
+the DateIntervalFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) applications) to manage and format date intervals.
+
+To manage date interval objects:
+
+1. Create corresponding object:
+
+    - DateInterval:
+     ```
```

Comment:
Global Comment:
Please intend the code according to the list.
![scr](https://user-images.githubusercontent.com/36875286/46646111-d29f4480-cb77-11e8-9c10-902405987989.JPG)


For example,
![ff](https://user-images.githubusercontent.com/36875286/46647089-a7b7ef00-cb7d-11e8-8713-1b9e8dd6c15e.JPG)